SPECS/binutils/binutils.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Contains a linker, an assembler, and other tools
 Name:           binutils
 Version:        2.36.1
-Release:        3%{?dist}
+Release:        4%{?dist}
```

Comment:
Please also update the following to reflect the new binutils and mariner-rpm-macros release numbers:
pkggen_core_x86_64.txt
pkggen_core_aarch64.txt
toolchain_x86_64.txt
toolchain_aarch64.txt
(all under: toolkit/resources/manifests/package)

Reply:
done

Reply:
done